.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 
 install:
 - gometalinter --install
-- sed -i 's/--deadline=50s/--deadline=120s/g'
+- sed -i 's/--deadline=50s/--deadline=180s/'
```

Comment:
just a small note: we can just create a patch to repo-infra with --deadline option

Reply:
TBH I wasn't fully aware of this, I've created a PR https://github.com/kubernetes/repo-infra/pull/75. Thanks 😄